sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileSourceStrategy.scala

```diff
@@ -177,6 +177,8 @@ object FileSourceStrategy extends Strategy with Logging {
       // Partition keys are not available in the statistics of the files.
       val dataFilters =
         normalizedFiltersWithoutSubqueries.filter(_.references.intersect(partitionSet).isEmpty)
+      logInfo(s"Pushed Filters: " +
+        s"${dataFilters.flatMap(DataSourceStrategy.translateFilter).mkString(",")}")
```

Comment:
We will run more than once translateFilter.

Reply:
Yes, but if we want to print the pushed filter once, I can not find a better way.

Reply:
I think run translateFilter twice is worse than logging twice.

Reply:
Before this pr, translateFilter is also run twice. This just reduce one logging.
How about pass pushedDownFilters in `case class FileSourceScanExec()` ?